tests/wpt/mozilla/tests/mozilla/mozbrowser/iframe_visibility.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<meta charset="utf-8">
+<head>
+<title>Iframe visibility setter/getter</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <script>
+    async_test(function(t) {
+        var expectedVisibilities = [false, true];
+        var receivedVisibilities = [];
+
+        var iframe = document.createElement("iframe");
+        iframe.mozbrowser = true;
+        iframe.src = "http://web-platform.test:8000";
+
+        //Alternate the iframe's visibility and ensure it's reflected by event/getter
+        iframe.onload = function() {
+            iframe.setVisible(false);
+            iframe.setVisible(true);
+        };
+
+        iframe.addEventListener("mozbrowservisibilitychange", e => {
+            assert_equals(iframe.getVisible(), e.detail.visible);
+            receivedVisibilities.push(e.detail.visible);
+            if (receivedVisibilities.length == expectedVisibilities.length) {
+                assert_array_equals(receivedVisibilities, expectedVisibilities);
+                t.done();
+            }
+        });
+
```

Comment:
Can we also test `requestAnimationFrame` and `setTimeout` behavior?
Also test that subiframes are correctly affected?